### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by Mzuha**

### CODE REVIEW

**Review by Veronika-Alekseienko — APPROVED** [review]
Well done!)

File: src/main/java/mate/academy/spring/controller/UserController.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        userService.add(john);
+        return "Done!";
+    }
+
+    @GetMapping("/{userId}")
+    public UserResponseDto get(@PathVariable Long userId) {
+        return userMapper.parse(userService.get(userId));
+    }
+
+    @GetMapping()
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/spring/controller/UserController.java:43]:
```suggestion
    @GetMapping
```

File: pom.xml [hunk 2]
```diff
@@ -72,6 +89,11 @@
                     <linkXRef>false</linkXRef>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>3.3.1</version>
+            </plugin>
```
Comment (KseniiaMakarova) [inline pom.xml:96]: does it work without `web.xml`?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (Mzuha) [reply pom.xml:96]: image
yes

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**